### DISCUSSION TITLE
err) Built-in CSS support is being disabled due to custom CSS configuration being detected.

### DISCUSSION THREAD

**Discussion by junhyeong32** [vercel/next.js#19011] [Help]
What can I do to avoid getting this error? Help me, please..

err :
Warning: Built-in CSS support is being disabled due to custom CSS configuration being detected.
See here for more info: https://err.sh/next.js/built-in-css-disabled

-package.json

```
"dependencies": {
    "@zeit/next-less": "^1.0.1",
    "@zeit/next-sass": "^1.0.1",
    "antd": "^4.8.1",
    "axios": "^0.21.0",
    "less": "^3.12.2",
    "mobx": "^6.0.3",
    "mobx-react-lite": "^3.1.5",
    "next": "10.0.1",
    "react": "17.0.1",
    "react-cookie": "^4.0.3",
    "react-dom": "17.0.1",
    "sass": "^1.29.0"
  }
```

-next.config.js

```
const withSass = require("@zeit/next-sass");
const withCSS = require('@zeit/next-css')
const withLess = require('@zeit/next-less')

if (typeof require !== 'undefined') {
    require.extensions['.less'] = file => { };
}

module.exports = withCSS({
    cssModules: true,
    cssLoaderOptions: {
        importLoaders: 1,
        localIdentName: "[local]___[hash:base64:5]",
    },
    ...withLess(
        withSass({
            cssModules: true,
            lessLoaderOptions: {
                javascriptEnabled: true,
            },
        })
    )
});

```

-Project structure

![스크린샷 2020-11-10 오후 7 16 42](https://user-images.githubusercontent.com/40843085/98661000-4af51700-2389-11eb-824d-51c044e9168c.png)

**Comment by jbryan11** [comment]
Well, that message is not an error but a warning wherein it prompts you that you're using a custom CSS config instead of built in CSS. In the link that you had given, It also says there that you can just ignore the message.

**Reply by shakesBeardZ** [reply]
nope it's an error, any CSS import won't work after that.

**Comment by shakesBeardZ** [comment]
@junhyeong32 did you manage to solve it? this message is indicating the CSS won't work anymore

**Reply by stigkj** [reply]
It is not an error, it is a warning. We have this warning in our project and the CSS imports work perfectly.

The warning is just saying you are using a custom way of handling CSS (i.e. `@zeit/next-css` in `next.config.js`) and not the built-in support that was added in Next.js v9.2.

It seems you have component-level CSS, so you should be able to use the built-in support for CSS: https://nextjs.org/docs/basic-features/built-in-css-support. This also supports Sass out of the box, while Less needs the plugin `@zeit/next-less` you already have in your `next.config.js` file.

**Reply by stigkj** [reply]
@junhyeong32 did you manage to get it to work?
